main.go

```diff
@@ -140,6 +140,27 @@ func main() {
 			Usage:   "Define safe directories",
 			EnvVars: []string{"PLUGIN_SAFE_DIRECTORY", "CI_WORKSPACE"},
 		},
+		&cli.BoolFlag{
+			Name:    "use-ssh",
+			Usage:   "Using ssh for git clone",
+			EnvVars: []string{"PLUGIN_USE_SSH"},
+			Value:   false,
+		},
+		&cli.StringFlag{
+			Name:    "ssh-key",
+			Usage:   "SSH key for ssh clone",
+			EnvVars: []string{"PLUGIN_SSH_KEY"},
+		},
+		&cli.StringFlag{
+			Name:    "ssh-user",
+			Usage:   "Username for ssh clone",
+			EnvVars: []string{"PLUGIN_SSH_USER"},
+		},
+		&cli.StringFlag{
+			Name:    "forge-url",
+			Usage:   "Forge URL for ssh clone",
+			EnvVars: []string{"CI_FORGE_URL"},
```

Comment:
Can we remove forge url and not cpnvert http to git url.

Instead use the new introduced env if `use-ssh: true`

Reply:
Sounds good to me